src/lib/LoginFile.js

```diff
@@ -102,28 +99,24 @@ class LoginFile {
 
     _drawWarningText() {
         const x = LoginFile.WIDTH / 2;
-        const y = LoginFile.HEIGHT - 32;
+        const y = LoginFile.HEIGHT - 86 - 2;
         this._ctx.fillStyle = 'white';
-        this._ctx.fillText('Do not share this file.', x, y);
+        this._ctx.fillText('Keep safe and confidential.', x, y);
     }
 
     /**
      * @param {string} encodedSecret
      */
     _drawQrCode(encodedSecret) {
-        const $el = document.createElement('div');
-        /* eslint-disable no-multi-spaces */
         const $canvas = QrEncoder.render({
             text: encodedSecret,
-            radius: 0.7,    // We encode 56 bytes. To keep within a smaller QR base size, we need to reduce
-            ecLevel: 'M',   // the error-correction level to M. Thus we reduce the radius from .8 to .7
-            fill: 'white',  // to reduce scanning issues.
+            radius: 0.5,
```

Comment:
Why did the radius decrease even more? Changing it from .8 to .7 was worth a comment. Why is it now .5 without the comment?

Reply:
Because it looks bad in such a high resolution. The gaps in the radiuses are very visible in high res.

Reply:
Actually .5 is the maximum allowed value.
Values over .5 look glitchy.

Reply:
Yes. It's not too bad with low resolution, as the missing pixels hide it somewhat, but sadly it became obvious now.